config/iam/recommended-policy-arn

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Can you remove this empty newline please? Not sure if it will mess up our scripts that read this file or not, but might as well not take the chance :)